### PR TITLE
Unittest for server File handler

### PR DESCRIPTION
**Pull request by meelsaw**
- added Testing script for FileHandler module
- added Print out option in requested format

### CODE REVIEW

File: src/server/main.py [hunk 1]
```diff
@@ -45,12 +78,16 @@ def receive_from_client(client_data):
     else:
         message = deserialised["INPUT_STRING"]
         if deserialised["PRINT_OUTPUT"] and deserialised["FILE_OUTPUT"]:
```
Comment (husky960) [inline src/server/main.py:80]: Aren't we already printing the output in the "print_out" function? Printing here again would result in repetition.

File: tests/server/FileHandler_test.py
Comment (husky960) [inline tests/server/FileHandler_test.py:1]: Looks good!

**Review by husky960 — CHANGES_REQUESTED** [review]
In general, looks good! Please make the minor changes on "main.py" before approval.

File: src/server/main.py (hunk 1 above)
Comment (meelsaw) [reply src/server/main.py:80]: Printing XML and Binary format is not readable by the user. That's why we're printing the original message then printing the formatted message.

File: src/server/main.py
[the remark this answers is not in the document]
Comment (meelsaw) [reply src/server/main.py:1]: Resolved

**Review by husky960 — APPROVED** [review]
LGTM!

**Pull request merged**